docs/features/Installation.md

```diff
@@ -45,28 +45,157 @@ The Common Gateway is a Common Ground application built from separate components
 
 ### Installation through Helm charts (recommended method)
 
-Before you continue, make sure you use Helm V3 for all the following steps.
+> **Dependencies**
+> - For installations on Kubernetes, you will need to install Helm v3 on your local machine.
+> - If you need to install LetsEncrypt, you will also need to install `kubectl`, but if you previously installed Docker Desktop then `kubectl` is already present on your machine (so no installation needed).
 
-And before installing the gateway through Helm charts, make sure you configure cert-manager on your cluster. For more information please visit the [cert-manager documentation](https://cert-manager.io/docs/).
+### Haven installations
+If you are installing the Common Gateway on a Haven environment, you can just run the provided Helm installer. Before installing the Common Gateway, you need to add it to your Helm repositories.
 
-Also, make sure to also install the nfs provisioner, see: [ArtifactHub](https://artifacthub.io/packages/helm/kvaps/nfs-server-provisioner). When installing nfs make sure you install with persistence enabled. And make sure to configure a persistence size higher than 1Gi (8Gi or even higher is recommended). The persistence size must be higher than the size of the gateway vendor Persistent Volume Claim(s combined) on your cluster.
```

Comment:
I think we are losing some information here, that I can't find anywhere else

Reply:
I'll re-add this information